runtime/ftplugin/astro.vim

```diff
@@ -0,0 +1,186 @@
+function! astro#IdentifyScope(start, end) abort
+    let pos_start = searchpairpos(a:start, '', a:end, 'bnW')
```

Comment:
As I remember, you can't just move functions from an autoload file.
```
function! astro#IdentifyScope(start, end) abort
```
Here, in the function name, the name of the autoload file is specified before the '#' character. Accordingly, they must be deleted.

But take your time, please, you need to check what the function does and whether it will work if you just move it from one file to another.